CommonCPack.cmake

```diff
@@ -45,7 +45,6 @@ set(CPACK_PACKAGE_VERSION ${VERSION})
 set(CPACK_RESOURCE_FILE_LICENSE ${PROJECT_SOURCE_DIR}/LICENSE.txt)
 if(NOT CPACK_PACKAGE_LICENSE)
   set(CPACK_PACKAGE_LICENSE "LGPL")
-  message(STATUS "Using default ${CPACK_PACKAGE_LICENSE} package license")
```

Comment:
Is this a good idea? I would rather make it a FATAL_ERROR, because declaring a license for a project automatically seems wrong.